xml/System/Guid.xml

```diff
@@ -1149,7 +1151,7 @@ After trimming, the length of the read-only character span is 0.
 ## Examples  
  The following example creates a new GUID, converts it to three separate string representations by calling the <xref:System.Guid.ToString%28System.String%29> method with the "B", "D", and "X" format specifiers, and then calls the <xref:System.Guid.Parse%2A> method to convert the strings back to <xref:System.Guid> values.  
   
- [!code-csharp[System.Guid.Parse#3](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.guid.parse/cs/parseex1.cs#3)]
+ [!code-csharp-interactive[System.Guid.Parse#3](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.guid.parse/cs/parseex1.cs#3)]
```

Comment:
this is missing the note. Add it to the MemberGroup element.

Reply:
I've added an INCLUDE above this example -- please see my comment for old line 1273 below.

Reply:
I've updated this Member to a MemberGroup & moved the INCLUDE to the ##Remarks section.